src/sentry/models/organizationmember.py

```diff
@@ -201,12 +205,11 @@ class OrganizationMember(Model):
     teams = models.ManyToManyField(
         "sentry.Team", blank=True, through="sentry.OrganizationMemberTeam"
     )
-    inviter = FlexibleForeignKey(
+    inviter_id = HybridCloudForeignKey(
         settings.AUTH_USER_MODEL,
         null=True,
         blank=True,
-        related_name="sentry_inviter_set",
-        on_delete=models.SET_NULL,
+        on_delete="SET_NULL",
     )
```

Comment:
Does this need a migration?

Reply:
@vbro That work was done in another pull request here https://github.com/getsentry/sentry/pull/48646.

You can safely ignore this pull request as I'm using this branch to run CI.